.circleci/config.yml

```diff
@@ -330,9 +319,9 @@ jobs:
       - run:
           name: install emsdk
           command: |
-            wget https://github.com/juj/emsdk/archive/master.tar.gz
-            tar -xf master.tar.gz
-            cd emsdk-master
+            wget -O ~/emsdk-master.tar.gz https://github.com/juj/emsdk/archive/master.tar.gz
+            tar -C ~ -xf ~/master-master.tar.gz
```

Comment:
Honestly tempted to suggest `wget -O - https://github.com/juj/emsdk/archive/master.tar.gz | tar -xz -C ~` just to avoid file name issues.